modules/hostap/CMakeLists.txt

```diff
@@ -58,20 +58,20 @@ zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_NO_DEBUG
   CONFIG_NO_STDOUT_DEBUG
 )
 
-zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_ROBUST_AV
-  CONFIG_ROBUST_AV
+zephyr_library_compile_definitions_ifndef(CONFIG_WIFI_NM_WPA_SUPPLICANT_ROBUST_AV
```

Comment:
I find it hard to follow the negated options. Would it be possible to use the `CONFIG_ROBUST_AV` in zephyr but then set the `CONFIG_NO_ROBUST_AV` for hostap only?

Reply:
Sure, we can do that.

Reply:
Wait, in Zephyr we already have a proper feature flags like `WIFI_NM_WPA_SUPPLICANT_ROBUST_AV`, it's just the hostap symbols we are defining  in Kconfig `NO_ROBUST_AV`, so, they need to match.

Reply:
Ok, I was just thinking that we could have both, meaning that if we have `ROBUST_AV` set, then we could unset the `NO_ROBUST_AV` and if it is not set, then we could set the `NO_ROBUST_AV`. But that might be confusing too.
At least for me, I always need mental acrobatics to figure out what is being set if there are these negated options. It is just more easier to read the code with normal positive options.

Reply:
Yeah, my original proposal was using positive flags but changed it to as per code review comments upstream. Let's leave it with double-negation `#ifndef + NO` to match upstream.

Reply:
Sure, no worries here.